_includes/selector.html

```diff
@@ -513,115 +501,86 @@
-                    "https://github.com/rapidsai/cuspatial/blob/branch-" + version + "/README.md",
-                    "https://github.com/rapidsai/cuxfilter/blob/branch-" + version + "/CONTRIBUTING.md",
-                    "https://github.com/rapidsai/cusignal#source-linux-os",
-                    "https://github.com/rapidsai/cucim/blob/branch-" + version + "/CONTRIBUTING.md#building-and-testing-cucim-from-source"
-                ];
-                var cmd = "<p class='selector__source-cmd'>For source build instructions, please view the links below:</p>";
-                cmd += "<ul class='selector__source-cmd__list'>";
-                for (var i = 0; i < links.length; i++) {
-                    var link = links[i];
-                    cmd += "<li class='selector__source-cmd__list-item'>";
-                    cmd += "<a class='selector__source-cmd__link' target='_blank' href='" + link + "'>" + link + "</a>";
-                    cmd += "</li>";
-                }
-                cmd += "</ul>";
-                return cmd;
-            },
-            getImgTypeText(type) {
-                if (type === "core") return "Basic"
-                if (type === "standard") return "Basic w/ Dask-SQL"
-                return type;
-            },
-            getImgOptionText(option) {
-                if (option === "notebooks") return "Include Jupyter Notebooks";
-                if (option === "development") return "Development Image (only on Docker Hub)";
-                return option;
-            },
             getImgLoc(loc) {
                 if (loc === "NGC") return 'NGC (Stable Only) <i class="fa-regular fa-share-nodes"></i>';
                 if (loc === "Dockerhub") return 'Docker Hub (Stable and Nightly) <i class="fa-docker fab"></i>';
                 return loc;
             },
             getDockerNotes() {
-                var core_pkgs = this.rapids_meta_pkgs;
+                var cuda12 = ""
+                if (this.active_cuda_ver.startsWith("12")) {
+                        var pkgs_html = this.rapids_meta_pkgs.filter(pkg => pkg !== "cuSignal").map(pkg => "<code>" + pkg + "</code>").join(", ");
+                        cuda12 = "RAPIDS Docker images for CUDA 12 do not currently support ARM architectures<br>";
+                } else {
+                        var pkgs_html = this.rapids_meta_pkgs.map(pkg => "<code>" + pkg + "</code>").join(", ");
+                    }
 
-                if (this.active_img_type === "standard") {
-                    core_pkgs = [...core_pkgs, "dask-sql"];
-                }
-                var pkgs_html = core_pkgs.map(pkg => "<code>" + pkg + "</code>").join(", ");
-                return [this.note_prefix + " The selected image contains the following packages:<br>" + pkgs_html];
+                return [this.note_prefix + cuda12 + " The selected image contains the following packages:<br>" + pkgs_html];
```

Comment:
These changes might need to be tweaked.

They introduce some visual differences (see screenshots below) for the Docker notes compared to the other notes.

Namely, the line and icon spacing is different.

To fix this, each note should be its own entry in the returned array.


<details><summary>Click to Expand - Docker Notes</summary>
<p>

![image](https://github.com/rapidsai/docs/assets/7400326/ccc525ce-ff82-46d1-9d10-bb0861f13a25)

</p>
</details> 

<details><summary>Click to Expand - Conda Notes</summary>
<p>

![image](https://github.com/rapidsai/docs/assets/7400326/856125f7-f847-470c-9877-51cd93caf985)

</p>
</details>

Reply:
Will make Docker notes match the Conda notes formatting

Reply:
Took a few tries because the VSCode preview I use doesn't quite match the committed version, but should be resolved in https://github.com/rapidsai/docs/pull/414/commits/5427ebf4e35b4dea52c211704dbd4dcbff53a370